CLAUDE.md

```
@@ -0,0 +1,92 @@
+- 깃 커밋/푸시는 사용자가 직접 수행.
+
+## 기술 스택
+- **Core**: Spring Boot 3.5.9 (Java 17), Gradle
+- **DB**: MySQL 8.0(User), MongoDB 6(Paper/Log), Redis 7(Cache), Elasticsearch 7.17(Search)
+- **Infra**: Docker Compose, Prometheus + Grafana, JWT Auth
+
+## 코드 컨벤션
+
+### 아키텍처 구조
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)